From: Felipe Sateler <fsateler@debian.org>
Date: Sun, 5 Apr 2020 12:10:36 -0400
Subject: Install rtkitctl to sbin

That is the path previously used in the autotools system
---
 meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/meson.build b/meson.build
index 50e137c..02e6c73 100644
--- a/meson.build
+++ b/meson.build
@@ -115,6 +115,7 @@ executable(
         'rtkitctl',
         'rtkitctl.c', 'rtkit.h', config_h,
         install: true,
+        install_dir: get_option('sbindir'),
         dependencies: [dbus_dep],
 )
 
